### PR TITLE
BRD - UI: Ux fixes

### PR DESCRIPTION
**Pull request by deetz99**
partial issue: bcGov/entity#23341

- updated with new core layer version
- add breadcrumbs
- several ux changes, padding, border radius, etc

### CODE REVIEW

File: business-registry-dashboard/app/components/Form/AddBusiness/Base.vue
Comment (deetz99) [inline business-registry-dashboard/app/components/Form/AddBusiness/Base.vue:1]: switch to component provided by layer

File: business-registry-dashboard/app/plugins/auth-api.ts
Comment (deetz99) [inline business-registry-dashboard/app/plugins/auth-api.ts:1]: plugin moved to layer

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-registry-dev--pr-27-ac484vev.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**